static/js/legal/Filters.js

```diff
@@ -0,0 +1,74 @@
+const FilterPanel = require('./FilterPanel');
+
+function Filters(props) {
+  const requestorOptions = [ {'value': 0, 'text': 'Any'},
+    {'value': 1, 'text': 'Federal candidate/candidate committee/officeholder'},
+    {'value': 2, 'text': 'Publicly funded candidates/committees'},
+    {'value': 3, 'text': 'Party committee, national'},
+    {'value': 4, 'text': 'Party committee, state or local'},
+    {'value': 5, 'text': 'Nonconnected political committee'},
+    {'value': 6, 'text': 'Separate segregated fund'},
+    {'value': 7, 'text': 'Labor Organization'},
+    {'value': 8, 'text': 'Trade Association'},
+    {'value': 9, 'text': 'Membership Organization, Cooperative, Corporation W/O Capital Stock'},
+    {'value': 10, 'text': 'Corporation (including LLCs electing corporate status)'},
+    {'value': 11, 'text': 'Partnership (including LLCs electing partnership status)'},
+    {'value': 12, 'text': 'Governmental entity'},
+    {'value': 13, 'text': 'Research/Public Interest/Educational Institution'},
+    {'value': 14, 'text': 'Law Firm'},
+    {'value': 15, 'text': 'Individual'},
+    {'value': 16, 'text': 'Other'}]
+
+  const documentTypes = [{'value' : 'F', 'text': 'Final Opinion'},
+   {'value' : 'V', 'text': 'Votes'},
+   {'value' : 'D', 'text': 'Draft Documents'},
+   {'value' : 'R', 'text': 'AO Request, Supplemental Material, and Extensions of Time'},
+   {'value' : 'W', 'text': 'Withdrawal of Request'},
+   {'value' : 'C', 'text': 'Comments and Ex parte Communications'},
+   {'value' : 'S', 'text': 'Commissioner Statements'}]
+
+  const resultCountChange = props.query.resultCount - props.query.lastResultCount;
+
+  return <div>
+            <TextFilter key="ao_no" name="ao_no" label="AO number" value={props.query.ao_no}
+                handleChange={props.setQuery} getResults={props.getResults} />
+            <TextFilter key="ao_requestor" name="ao_requestor" label="Requestor Name (or AO Name)" value={props.query.ao_requestor}
+                  handleChange={props.setQuery} getResults={props.getResults} />
+              <Dropdown key="ao_requestor_type" name="ao_requestor_type" label="Requestor Type" value={props.query.ao_requestor_type}
+                options={requestorOptions} handleChange={props.instantQuery} />
+            <ul className="accordion--neutral" data-content-prefix="first">
+              <FilterPanel id="first-content-0" header="Documents" startOpen={true}
+                  content={[<TextFilter key="q" name="q" label="Keywords" value={props.query.q}
+                  handleChange={props.setQuery} getResults={props.getResults} />,
+                <CheckboxFilter key="ao_is_pending" name="ao_is_pending" label="Show only pending requests"
+                    checked={props.query.ao_is_pending} handleChange={props.instantQuery} />,
+                <CheckboxList key="ao_category" name="ao_category" label="Document Type" value={props.query.ao_category || ['F']}
```

Comment:
These `<CheckboxList>` components seem to be at a higher level of indentation as the `<FilterPanel>`, which made me think they were children of it, but they seem to be siblings... maybe reduce their indentation level?